### PR TITLE
Update brand logo margin

### PR DESCRIPTION
**Pull request by pkong-ds**
ref DEV-2016

portal height slider in next PR.

## Preview

https://github.com/user-attachments/assets/30eca53a-eb5a-4543-9bfd-62aa99981429

### CODE REVIEW

File: resources/authgear/templates/en/web/authflowv2/__brand_logo.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
     </a>
   </div>
 {{ else }}
-  <div class="brand-logo__container light">
+  <div class="brand-logo__container light" data-src="{{ $light }}">
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/web/authflowv2/__brand_logo.html:48]: Copy data-previewable-*

File: authui/src/authflowv2/components/settings-header.css [hunk 2]
```diff
@@ -8,20 +8,11 @@
   }
 
   .settings-header {
-    @apply hidden;
+    @apply hidden; /* NOTE(DEV-2123): Hide setting header for now */
```
Comment (louischan-oursky) [inline authui/src/authflowv2/components/settings-header.css:11]: Remove this commit.

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Comment by pkong-ds** [comment]
Updated thanks!

**Comment by pkong-ds** [comment]
Thanks 🙏

**Comment by louischan-oursky** [comment]
@pkong-ds I added 3 commits.

**Comment by pkong-ds** [comment]
@louischan-oursky I did not use gap in screen-logo-layout because it's not the only place that uses brand logo.

I believe __direct_access and __settings_header both use it. If the margin is removed without adding gap to these 2 files, the existing UI will probably break.

Maybe we should add gap to those 2, or keep margin bottom?

**Comment by pkong-ds** [comment]
Also, curious, is object contain really necessary with flex?

**Review by louischan-oursky — APPROVED** [review]

**Comment by pkong-ds** [comment]
Per offline discussion

> __direct_access and __settings_header both use it

__direct_access - Fixed by @louischan-oursky in [cf53deb](https://github.com/authgear/authgear-server/pull/4767/commits/cf53debc8ad9a493f3abbf19ac4b8eb99b3a3123), [42ef98b](https://github.com/authgear/authgear-server/pull/4767/commits/42ef98b1a2954531ad15f083bfb30f842ea1af7f)
__settings_header - to be removed in PR #4770 

> is object contain really necessary

Turns out it was a bug previously, @louischan-oursky helped fix it in [b5224dc](https://github.com/authgear/authgear-server/pull/4767/commits/b5224dc9e65abd5b4cc7f607bf1285969324fa2e)

**Pull request merged**